src/Scripting.cpp

```diff
@@ -234,9 +234,14 @@ static cell AMX_NATIVE_CALL SetFakeFacingAngle(AMX *amx, cell *params)
 	if(!IsPlayerConnected(playerid))
 		return 0;	
 
+	if (Player::fakeQuat[playerid])
```

Comment:
This is not needed at all, deleting a null pointer is well defined.

Reply:
> This is not needed at all, deleting a null pointer is well defined.

You're right, that can be removed.